indico/htdocs/sass/modules/_sessions.scss

```diff
@@ -0,0 +1,48 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+@import 'base';
+@import 'compass';
+@import 'partials/icons';
+
+.sessions {
+    thead {
+        th {
+            &:not(.sorter-false) {
+                cursor: pointer;
+            }
+
+            &:before {
+                @include transition(transform 0.3s ease-out);
+                -webkit-transform-origin: 50%  51%; // improve blurry text on rotate with chrome
```

Comment:
this is the 5th incarnation of this line in the indico codebase . can we somehow avoid repeating it all the time? ping @OmeGak